README.md

```diff
@@ -32,6 +32,14 @@ information on how to run them.
 
 Set up a PostreSQL database. One option is to run a [Postgres Docker image][postgres-docker].
 
+#### Docker Setup
+
+When setting up the docker container ensure the run command is as follows, including the port information after the -d argument:
```

Comment:
I think the important thing is for the port argument to be before the container name, rather than specifically after the `-d` argument.

Reply:
Fixed in new commit, see below: https://github.com/nationalarchives/tdr-prototype-sangria/pull/10/commits/5bcc46fd27ba83a3c6083d41358e658d8cbb4493